sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/ConfigUtil.java

```diff
@@ -139,6 +140,12 @@ private static ClassLoader getClassLoader() {
         return classLoader;
     }
 
+    private static Charset getCharset() {
+        // avoid static loop dependencies: SentinelConfig -> SentinelConfigLoader -> ConfigUtil -> SentinelConfig
+        // so not use SentinelConfig.charset()
+        return Charset.forName(System.getProperty(SentinelConfig.CHARSET, StandardCharsets.UTF_8.name()));
```

Comment:
We should not reference `SentinelConfig` here, or the circular dependency problem will still exist. Maybe we could just copy the constant, or move the constant to elsewhere.

Reply:
`ConfigUtil` only references the constants of `SentinelConfig`, does not trigger the initialization of SentinelConfig, i think no circular dependency here.

Reply:
I understand what you mean now, I think copy constants is the best way.